gui_qt/app.py

```
@@ -1064,6 +1078,26 @@ def _same_manifest_path(self, left: str, right: str) -> bool:
             return False
         return canonical_abs_path(left).lower() == canonical_abs_path(right).lower()
 
+    def _split_entries_unchanged(
+        self,
+        entries: list[SplitManifestEntry],
+    ) -> bool:
+        previous = self._split_status_entries
+        if len(previous) != len(entries):
+            return False
+        for old_entry, new_entry in zip(previous, entries):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)